rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -241,10 +241,12 @@ def add_postprocessed_batch_for_training(
                 training).
         """
         for view_col, data in batch.items():
+            # TODO(ekl) how do we handle this for policies that don't extend
+            # Torch / TF Policy template (no inference of view reqs)?
             # Skip columns that are not used for training.
-            if view_col not in view_requirements or \
-                    not view_requirements[view_col].used_for_training:
-                continue
+            # if view_col not in view_requirements or \
+            #         not view_requirements[view_col].used_for_training:
+            #     continue
```

Comment:
@sven1977 I think we have to drop this for policies that don't go through the policy template. Is there any strong reason not to drop this?